tests/lib/rules/jsx-indent.js

```diff
@@ -159,6 +188,15 @@ ruleTester.run('jsx-indent', rule, {
       '  );',
       '}'
     ].join('\n'),
+    output: [
+      'function App() {',
+      '  return (',
+      '    <App>',
+      '  <Foo />',
+      '</App>',
+      '  );',
+      '}'
+    ].join('\n'),
```

Comment:
I'd rather leave the broken but correct test, and comment it out, than leave this incorrect test. Could we do that?